dist/adaptive-html.cjs.js

```diff
@@ -625,7 +625,7 @@ TurndownService.prototype = {
     var _this = this;
 
     var currText = '';
-    var blocks = Array.prototype.reduce.call(parentNode.childNodes, function (output, node) {
+    var blocks = Array.prototype.reduce.call(parentNode.childNodes || [], function (output, node) {
```

Comment:
what is this javascript witchery

Reply:
its a short hand to resolve objects if `parentNode.childNodes` is null or undefined then the statement resolves as

```
null || []
```

which will return `[]` in it's place